pkg/server/api_v2_databases_metadata.go

```diff
@@ -323,7 +324,9 @@ func (a *apiV2Server) getTableCreateStatement(
 	row, types, err := a.sqlServer.internalExecutor.QueryRowExWithCols(ctx, "get-table-create-statement", nil,
 		sessiondata.NodeUserSessionDataOverride, query.String(), query.QueryArguments()...)
 	if err != nil {
-		return "", err
+		statementError := fmt.Sprintf("Unable to retrieve create statement for %s.%s", escDbName, escTableName)
+		log.Warningf(ctx, "%v", errors.Wrapf(err, "%s", statementError))
+		return statementError, nil
```

Comment:
nit: I don't like mixing error message with response string. Ideally this should be returned as an error, and the client should render accordingly. This way the client has no way to know that the create statement is not actually present. I know the whole point in this PR was to actually not return an error, but we should be able to differentiate between error types in the client. something to consider moving forward.